server/users/index.js

```diff
@@ -152,4 +152,23 @@ router.post('/:userId/password-reset', async (req, res, next) => {
   }
 });
 
+//chat rooms
+
+router.listChatHistory = (helperP = helpers, gameId = process.env.CHAT_ROOM_GAME_ID) => (req, res, next) => {
```

Comment:
Let's call this `CHAT_ROOM_PREFIX`: it's more generic. Will be more natural in case (for example) we would reuse this admin interface with another project that doesn't share the same convention to use a game ID as a prefix, or isn't a game at all.